scrapers/SexSelector.yml

```
@@ -0,0 +1,35 @@
+name: "SexSelector"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sexselector.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="player__info__top"]/h3/text()
+      Details: //p[@class="player__description"]/text() 
+      Date:
+        selector: //div[@class="player"]/p[starts-with(., "Released:")]
+        postProcess:
+          - replace:
+              - regex: "^Released: "
+                with:
+          - parseDate: Jan 2, 2006
+      Image:
+        selector: //a[@class="header__menu header__menu--logo"]/@href
+        postProcess:
+          - replace:
+              - regex: .+(pg\d+)
+                with: $1/poster/${1}_01_2160.jpg
+              - regex: ^
+                with: "https://sm-members.bangbros.com/shoots/sexselector/"
```

didn't notice this on discord, but there's no reason not to have it be:
```suggestion
              - regex: .+(pg\d+)
                with: https://sm-members.bangbros.com/shoots/sexselector/$1/poster/${1}_01_2160.jpg
```